### ISSUE TITLE
Motherless does not work

### ISSUE THREAD

**Issue by Poorrichardbar** [dobbelina/repository.dobbelina#1285]
### Prerequisites

- [X] I have searched the issue tracker to avoid opening a duplicate issue.
- [X] I have read the [documentation](https://github.com/dobbelina/repository.dobbelina/wiki/Cumination:-Debug-Logging-and-Uploading) on how to create a debug log file.

### Description

Unable to pull log due to fire stick.  When trying to use motherless add in it says site has a cloudflare challenge.  Have reloaded motherless add in and same reaults

### Kodi version

20.4

### Cumination version

1.1.100

### Operating System and version

Firestick

**Comment by Gujal00** [comment]
It is saying site has cloudflare challenge. So you cant use without running FlareSolverr. You can run FlareSolverr on a firestick, so have to run FlareSolverron some other machine on your localnetwork. If you cant, then no solution until motherless turn off cloudflare in your region

**Comment by dobbelina** [comment]
I tried it with FlareSolverr enabled but get no thumbnails.

**Comment by Gujal00** [comment]
@dobbelina Yes cant get thumbnails when server is behind cflare. need to append useragent and cookie returned by flaresolverr to thumbnails for it to show.  The UA is stored in settings so can get it from there but not sure about cookies. @ErosVece may have better idea
